### PR TITLE
Fixed timestamp string conversion error for cstmt

### PR DESCRIPTION
**Pull request by tkyc**
Fixes https://github.com/microsoft/mssql-jdbc/issues/2436.

### CODE REVIEW

File: src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java [hunk 1]
```diff
@@ -1294,6 +1309,15 @@ private static void createProcedureManyParams() throws SQLException {
         }
     }
 
+    private static void createProcedureCurrentTime() throws SQLException {
+        String type = manyParamUserDefinedType;
```
Comment (Jeffery-Wasty) [inline src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java:1313]: Unused
```suggestion
```

File: src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java [hunk 2]
```diff
@@ -1294,6 +1309,15 @@ private static void createProcedureManyParams() throws SQLException {
...
+        String sql = "CREATE PROCEDURE " + currentTimeProc + "@currentTimeStamp datetime = null OUTPUT " +
+                "AS BEGIN SET @currentTimeStamp = CURRENT_TIMESTAMP; END";
```
Comment (Jeffery-Wasty) [inline src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java:1315]: Space needed at start of second string?
```suggestion
        String sql = "CREATE PROCEDURE " + currentTimeProc + " @currentTimeStamp datetime = null OUTPUT " +
                "AS BEGIN SET @currentTimeStamp = CURRENT_TIMESTAMP; END";
```

**Review by Jeffery-Wasty — COMMENTED** [review]
Also, formatter has some suggestions for `CallableStatementTest.java`.

File: src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java (hunk 2 above)
Comment (lilgreenbird) [reply src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java:1315]: hmm how could this have worked?

Comment (lilgreenbird) [reply src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java:1315]: ok looks like the tests are failing due to this

**Review by lilgreenbird — APPROVED** [review]

**Review by Jeffery-Wasty — APPROVED** [review]

**Pull request merged**